content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -195,6 +195,13 @@ authentication is needed.
                  gdrive_service_account_json_file_path path/to/file.json
    ```
 
+   Alternatively, a `GDRIVE_CREDENTIALS_DATA` can be set to pass service account
+   key in CI/CD systems, production setup, read-only file systems, etc. The
+   content of this variable should be a string with JSON that has the same
+   format as in the keys file described above. If both this variable and
+   `gdrive_service_account_json_file_path` are provide, DVC reads the
+   environment variable first.
```

Comment:
I think that providing `gdrive_service_account_json_file_path` is no longer required after https://github.com/iterative/dvc/pull/7213

Reply:
> I read the section as saying that either gdrive_service_account_json_file_path or GDRIVE_CREDENTIALS_DATA is required.

I understood the phase wrong 🙏

Reply:
@dberenbaum @daavoo thanks! Done, I've changed the message a bit - pls take a look.

Reply:
> Finally, should we promote GDRIVE_CREDENTIALS_DATA as the primary method 

Make sense, but let's keep it as is for now.  Service accounts can be used (and are used) locally as well ...